src/fleetlearning/common/static_params.py

```diff
@@ -62,9 +62,9 @@ def __init__(self):
         self.BATCH_SIZE = 8
         self.VAL_FACTOR = 0.1  # percentage of train data to use for validation
         self.SUBSET_FACTOR = 0.003  # subset of test frames to use
-        self.USE_GPU = True
-        self.NUM_GLOBAL_ROUNDS = 3
-        self.NUM_LOCAL_EPOCHS = 4
+        self.USE_GPU = False
```

Comment:
I think this should be defaulted to True, or? That's at least how the tests are right now. but we could change them if that makes more sense. 
```suggestion
        self.USE_GPU = True
```

Reply:
same here, I forgot to reset these!